src/openforms/js/components/admin/form_design/FormStepDefinition.js

```diff
@@ -104,6 +104,32 @@ const FormStepDefinition = ({
     }
   );
 
+  const duplicateKeys = getDuplicatedComponents(componentNamespace)
+    .filter(component =>
+      configuration.components.some(configComponent => configComponent.key === component.key)
+    )
+    .map(component => {
+      const componentLocations = getComponentLocations(formSteps, component);
+      const locations = componentLocations.map(location => {
+        return `${location.name} > ${component.label}`;
+      });
+      return `"${component.key}" (in ${locations.join(', ')})`;
```

Comment:
this is not translatable, please use `intl.formatMessage(...)` for this so that we can properly localize/translate this.